zerver/lib/markdown/fenced_code.py

```diff
@@ -215,6 +243,38 @@ def check_for_new_fence(
         output.append(line)
 
 
+class KrokiHandler(BaseHandler):
+    def __init__(
+        self,
+        processor: Any,
+        output: MutableSequence[str],
+        fence: str,
+        lang: str,
+    ) -> None:
+        self.processor = processor
+        self.output = output
+        self.fence = fence
+        self.lang = lang
+        self.lines: List[str] = []
+
+    def handle_line(self, line: str) -> None:
+        self.lines.append(line)
+
+    def done(self) -> None:
+        text = "\n".join(self.lines)
+        processed_lines = text.rstrip()[:-3].rstrip()
```

Comment:
Generally we avoid numbers like the `3` here that could be `len("string_to_strip")`